test/unit/messaging/process-return-message.test.js

```diff
@@ -1,25 +1,91 @@
 jest.mock('ffc-messaging')
+
 const processReturnMessage = require('../../../app/messaging/process-return-message')
+
+jest.mock('../../../app/inbound/process-return-settlement')
+const processReturnSettlement = require('../../../app/inbound/process-return-settlement')
+
 let receiver
 let settlement
+let message
 
 describe('process return message', () => {
   beforeEach(() => {
+    processReturnSettlement.mockReturnValue(undefined)
     settlement = JSON.parse(JSON.stringify(require('../../mock-settlement')))
     receiver = {
       completeMessage: jest.fn()
     }
+    message = { body: { settlement } }
   })
 
   afterEach(() => {
     jest.clearAllMocks()
   })
 
-  test('completes message', async () => {
-    const message = {
-      body: { ...settlement }
+  test('should call processReturnMessage when nothing throws', async () => {
+    await processReturnMessage(message, receiver)
+    expect(processReturnSettlement).toHaveBeenCalled()
+  })
+
+  test('should call processReturnSettlement once when nothing throws', async () => {
+    await processReturnMessage(message, receiver)
+    expect(processReturnSettlement).toHaveBeenCalledTimes(1)
+  })
+
+  test('should call processReturnSettlement with settlement when nothing throws', async () => {
+    await processReturnMessage(message, receiver)
+    expect(processReturnSettlement).toHaveBeenCalledWith({ settlement })
```

Comment:
would remove curly braces: `toHaveBeenCalledWith(settlement)`

`settlement` is an object and in my head `{settlement}` is an object of an object but it isn't

Reply:
*gasp* I've just seen this is a copy-pasta from me -- I am my own enemy!?!!!!

Reply:
bahahaa brilliant